Added some bit library tests

FirstLemon opened the pull request:
Tried to add some tests for bit library
- `arshift`
- `band`
- `bnot`
- `bor`
- `bswap`
- `bxor`
- `lshift`
- `rol`
- `ror`
- `rshift`
- `robit`
- `tohex`

FirstLemon committed:
bit: providing the tests

sarahsturgeon reviewed (commented):
I'll give this a more thorough review when I free up, but on first glance, I'm impressed! 🚀 

Thank you :) 🫶

sarahsturgeon reviewed (approved):
LGTM

Two nits, I'll let you decide if you want to address them
1. Prefer `errWith( [[full error message]] )` over `.err()`
2. Keep the test names consistent (are you going to prefix with "Should"? keep casing consistent, etc.)

Thanks again!

FirstLemon committed:
bit: cleanup and fix

- fixed formatting issues
- hopefully made the names better
- corrected the tests a bit

FirstLemon committed:
Merge branch 'main' into bit-tests-pr

sarahsturgeon reviewed (approved):
So sorry I let this sit for so long! I took a bit of a hiatus. I sincerely appreciate the contribution - I hope you had a good experience with the tools. Please be sure to @ me with any frustrations or shortcomings (if you remember any, still!) ❤️

sarahsturgeon committed:
Merge branch 'main' into bit-tests-pr

sarahsturgeon committed:
Merge branch 'main' into bit-tests-pr
